utils/src/config.rs

```diff
@@ -136,25 +119,24 @@ pub struct ServerTimeout {
     #[serde(with = "duration_format", default = "default_rpc_timeout")]
     rpc_timeout: Duration,
 
-    /// How long a candidate should wait before it starts another round of election
+    /// How many ticks a candidate needs to wait before it starts a new round of election
+    /// It should be larger than `follower_timeout`
     #[getset(get = "pub")]
-    #[serde(with = "duration_format", default = "default_candidate_timeout")]
-    candidate_timeout: Duration,
+    #[serde(default = "default_candidate_timeout_ticks")]
+    candidate_timeout_ticks: u8,
```

Comment:
Is it necessary to split election_timeout_tick into two parts, the `candidate_timeout_tick` and the `follower_timeout_tick`?

Reply:
See below